src/__tests__/TTTComponent.test.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import TicTacToe from '../components/TicTacToe';
+
+describe('TTT component', () => {
```

Comment:
As silly as it might seem, I would spell this out. If you're testing a component named `TicTacToe`, this file should be `TicTacToe.test.tsx`

Reply:
"Don't make me think"